### DISCUSSION TITLE
Dark Theme

### DISCUSSION THREAD

**Discussion by barepixels** [lllyasviel/Fooocus#327] [General]
To view Fooocus dark mode, change your URL to  "http://127.0.0.1:7860/?__theme=dark"

![dark](https://github.com/lllyasviel/Fooocus/assets/92839033/e2af77b4-4aea-4a89-984e-c0466bb9abe1)

**Comment by Eldobaz** [comment]
Maybe this can be written to some configuration file so that it doesn't have to be entered manually every time?

**Reply by Marduc666** [reply]
I just added this in the .bat file at the end.
--theme dark

`.\python_embeded\python.exe -s Fooocus\entry_with_update.py --preset realistic --theme dark`

**Comment by metercai** [comment]
The dark mode can be switched through the topbar in this version: https://github.com/metercai/SimpleSDXL
![SimpleSDXL_en](https://github.com/lllyasviel/Fooocus/assets/5652458/886fb672-6c28-438f-a40a-c7af3ad88bd7)

**Comment by Lennyblonde** [comment]
awesome it worked :-) Thanx
